doc/source/getting_started/write_documentation.rst

```diff
@@ -0,0 +1,224 @@
+
+#. **Troubleshooting and FAQs**: Anticipate common issues and provide solutions
+   in a troubleshooting section. Frequently Asked Questions (FAQs) can also be
+   helpful for addressing common queries.
+
+#. **Maintain and Update**: Keep your documentation up-to-date as the project
+   evolves. New features, changes, and bug fixes should be reflected in the
+   documentation.
+
+#. **Solicit Feedback**: Invite users and contributors to provide feedback on
+   the documentation, and be responsive to their suggestions and questions.
+
+
+Vale linting tool
+=================
+
+On the GitHub repository, CICD runs `Vale <vale_>`_, a powerful and extensible linting tool for
+checking the writing of each pull request.
+If you want to verify locally as well, you must install the tool locally:
+
+Installation
+------------
+
+#. **Install Vale**: Follow the instructions given in `Installation <vale_installation_>`_
+#. **Verify Installation**: To confirm that Vale is installed correctly, run:
+
+   .. code:: console
+    
+      vale --version
+
+   You should see the installed Vale version displayed in the terminal.
+
+Usage
+-----
+
+Vale is a versatile tool for linting and style checking your documents,
+supporting various file formats and providing a wide range of style guides.
+Here's a basic example of how to use Vale in PyMAPDL:
+
+#. **Sync styles**: First time you run Vale in a repository you need to
+   sync the styles specified in the :file:`.vale.ini` file running:
+
+   .. code:: console
+
+      vale sync
+
+
+#. **Lint Your Document**: To verify a document, run Vale from the command line,
+   specifying the file or directory you want to lint. For example:
+
+   .. code:: console
+
+       vale --config="./doc/.vale.ini" path/to/your_document.rst
+
+   Vale analyzes your document, and if there are any style guide violations
+   or linting issues, it provides feedback in the terminal.
+
+Make sure you have no errors or warnings before opening your pull request.
+
+
+.. _ref_building_example:
+
+Create an example
+=================
+There are three types of examples: dynamic, static, and semi-static.
+
+* `Dynamic examples`_
+* `Static examples`_
+* `Semi-dynamic examples`_
+
+
+Dynamic examples
+----------------
+
+The dynamic examples are based on Python files and must be able to run in under three minutes.
+
+They are in the `examples <pymapdl_examples_>`_ directory in this repository.
+
+.. vale off
+
+Example: `2d_plate_with_a_hole.py <pymapdl_2d_plate_with_a_hole_>`_
+.. vale on
+
+Here is a link to this dynamic example: 
+`MAPDL 2D Plane Stress Concentration Analysis <pymapdl_doc_2d_plate_with_a_hole_>`_
+
+When an example is executed, **Total running time of the script** appears at the end of
+the document.
+As dynamic examples must run each time documentation is built, make sure that they are very short.
+To get around the problem of execution time, feel free to use static or semi-static examples.
```

Comment:
```suggestion

Because dynamic examples must run each time the documentation is built, make sure that they are very short.
To get around the problem of execution time, feel free to use static or semi-static examples.
```